src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_create_loadbalancer_then_update.yaml

```diff
@@ -549,29 +622,333 @@ interactions:
       - --resource-group --name --ssh-key-value --load-balancer-sku --load-balancer-managed-outbound-ip-count
         --load-balancer-outbound-ports --load-balancer-idle-timeout
       User-Agent:
-      - AZURECLI/2.49.0 azsdk-python-azure-mgmt-containerservice/24.0.0 Python/3.8.10
-        (Linux-5.15.0-1039-azure-x86_64-with-glibc2.29)
+      - AZURECLI/2.56.0 azsdk-python-azure-mgmt-containerservice/29.0.0 Python/3.10.13
+        (Linux-6.2.0-1018-azure-x86_64-with-glibc2.31)
+    method: GET
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.ContainerService/managedClusters/cliakstest000002?api-version=2023-11-01
```

Comment:
This PR adds 2 `GET` requests and 1 `PUT` request for `https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.ContainerService/managedClusters/cliakstest000002`

#28407 (merged on 2024-02-21) bumped the API version of `/providers/Microsoft.ContainerService/managedClusters` from `2023-11-01` to `2024-01-01` in the existing requests.

This PR's CI was run on 2024-01-30 and passed, but it is not re-run after #28407. As adding new lines won't cause merge conflict, this PR can be merged, but triggers CI failure after merging.